tests/pyccel/scripts/valid_assert_test.py

```diff
@@ -0,0 +1,11 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+
+if __name__ == "__main__":
+    assert True
+    a = 0
+    b = a
+    assert a == b
+    b = 1
+    assert a != b
+    assert a <= b
+    assert b >= a
```

Comment:
Tests are failing due to the name of this file. See the [pytest documentation](https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery):

> pytest implements the following standard test discovery:
> - search for test_*.py or *_test.py files, imported by their [test package name](https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#test-package-name).

If you rename this file so pytest doesn't think it contains a test for it to run then you should avoid this problem

Reply:
Oh i see, i got me so confused as to why pytest tries to import that file.